### PR TITLE
Eat through the explorer mask. Better explorer gear descriptions.

### PR DESCRIPTION
**Pull request by PerishedFraud**
## About The Pull Request

 This lets you eat through the explorer gas mask and removes the toggle made redundant by the change. On the side, it also changes its description and the explorer suit/hood description.

**History/notes:** This change was already approved and merged once though it kept the mask toggle. Ivanmixo would then go on to revert the change without citing a reason. They'd later reveal it was to maintain realism. I will take a moment to note that we have other masks that allow eating, as well as wizards. With this in mind, here's a better version of the change. 

This PR originally also contained jetpack changes, but those are now pending as a [poll](https://forums.beestation13.com/t/jetpacks-for-exploration-mining-crew-opinions-needed) is ongoing.

## Why It's Good For The Game

  

Mask change: Miners are the only crew members that regularly harvest, make and eat food while being on internals 24/7. Out of all crew, it makes the most sense for them to have masks with this feature. There's no downside to giving the masks this feature, and it will also make miners stop replacing their masks with welding gas masks specifically so they can eat easier, which is steadily rising in the miner meta. 

In conclusion, this change will be appreciated by the respective playerbase, as shown by their current actions, and doesn't bring anything even potentially negative to the game.

Descriptions: The mask doesn't reference the military anymore, and the suit upgrading is made more obvious.

## Changelog
:cl:
tweak: tweaked a few things
/:cl:

### CODE REVIEW

**Review by PowerfulBacon — COMMENTED** [review]
note that the component for armor plating automatically adds to the description.
`It can be strengthened with up to [maxamount] [upgrade_name].`

**Comment by PerishedFraud** [comment]
Noted and changed accordingly.

**Comment by BrumalLake** [comment]
2 prs and 3 forks later
why is it absolutely necessary to change explorer gear descriptions when they have the plate info already

**Comment by PerishedFraud** [comment]
It isn't. It's just useful. The mask description hints that it offers actual protection. The suit description makes its upgrading function more obvious. The descriptions were never the main focus of the PR though, and the amount of shenanigans behind it comes from the jetpack debate, which we have a poll for now.

**Comment by KubeRoot** [comment]
I want to note, the main reason I liked being able to eat through explorer mask was to take pills without taking off internals on lavaland. It allows you to have bottles of healing pills on you that you can apply with no delay, unlike patches, to heal in tough situations, even in the middle of combat.

**Comment by Archanial** [comment]
I don't like it
Mask even has special sprite for when you can't eat with it

**Comment by Archanial** [comment]
🔑

**Comment by ike709** [comment]
🔑

**Comment by Boxlta** [comment]
ike has 1984ed this pr

**Comment by ike709** [comment]
> ike has 1984ed this pr

If I wanted to 1984 it I'd ask GitHub Support to nuke it from existence.
